packages/server/src/api/controllers/query/index.ts

```diff
@@ -17,10 +17,12 @@ import {
   QueryPreview,
   QuerySchema,
   FieldType,
-  type ExecuteQueryRequest,
```

Comment:
Why removing the type keyword?

Reply:
@mike12345567 asked about them in another PR and I got rid of them there, so I did the same here. Not really any good reason other than that. It wasn't causing problems.

Reply:
I think its purely for dropping the import at runtime, but types is always included anyway, especially with using esbuild fairly sure the `type` keyword is meaningless.